### PR TITLE
docs: Update links to igraph webpage

### PR DESCRIPTION
**Pull request by olivroy**
I assumed the igraph homepage was the general igraph website, igraph.org, so I changed that. I wouldn't mind reverting to r.igraph.org if that's what you meant.

I changed the link to https://igraph.org/r/html/latest/ since it was documentation of version 1.3.5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ See the package wiki for more information on [installation troubleshooting](http
 
 ## Documentation
 
-See the [igraph homepage](https://igraph.org/r/) for the complete manual.
+See the [igraph homepage](https://igraph.org/) for the complete manual.
```
Comment (maelle) [inline README.md:52]: @krlmlr I think we are referring to the package manual here, so I'd rather rephrase the sentence than change the link.

File: DESCRIPTION [hunk 2]
```diff
@@ -34,7 +34,7 @@ Suggests:
 LinkingTo: 
     cpp11 (>= 0.2.0)
 License: GPL (>= 2)
-URL: https://r.igraph.org/, https://igraph.org/, https://igraph.discourse.group/
+URL: https://r.igraph.org/, https://igraph.org/, https://igraph.discourse.group/, https://github.com/igraph/rigraph/
```
Comment (maelle) [inline DESCRIPTION:37]: good catch!

File: vignettes/igraph_ES.rmd [hunk 3]
```diff
@@ -26,7 +26,7 @@ Para instalar la librería desde CRAN, usa:
 install.packages("igraph")
 ```
 
-Encontrarás más información sobre dependencias, requisitos y resolución de problemas sobre la instalación en la [página principal](https://igraph.org/r/).
+Encontrarás más información sobre dependencias, requisitos y resolución de problemas sobre la instalación en la [página principal](https://igraph.org/).
```
Comment (maelle) [inline vignettes/igraph_ES.rmd:29]:
```suggestion
Encontrarás más información sobre dependencias, requisitos y resolución de problemas sobre la instalación en la [página principal](https://r.igraph.org/).
```

**Review by maelle — COMMENTED** [review]
Thank you!!

**Comment by olivroy** [comment]
Thanks!

Just a reminder that the GitHub homepage still shows the old link as well!

image

**Comment by ntamas** [comment]
Fixed the link on the repo homepage, thanks!

**Review by ntamas — APPROVED** [review]

**Comment by krlmlr** [comment]
I removed the link to https://github.com/igraph/rigraph/, it's not typically added to the `URL` component.

File: README.md (hunk 1 above)
Comment (maelle) [inline README.md:52]:
```suggestion
See the [igraph package's website](https://igraph.org/) for the complete manual.
```

File: README.md [hunk 4]
```diff
@@ -49,7 +49,7 @@ See the package wiki for more information on [installation troubleshooting](http
 
 ## Documentation
 
-See the [igraph homepage](https://igraph.org/r/) for the complete manual.
+See the [igraph package's website](https://igraph.org/) for the complete manual.
```
Comment (szhorvat) [inline README.md:52]: Let's not call https://igraph.org/ the "package website". That would be https://r.igraph.org/, no?

**Comment by krlmlr** [comment]
@olivroy: Would you like to take another look?

**Comment by olivroy** [comment]
Thanks, changed the link in README to redirect to the R package's homepage r.igraph.org, per maelle and szhorvat's suggestions

Ready to merge!

**Comment by krlmlr** [comment]
Thanks!

**Pull request merged**